tests/unit/test_asynchronous_get.py

```diff
@@ -1,5 +1,6 @@
 import pytest
 from aleph_message.models import MessageType, MessagesResponse
+import unittest
```

Comment:
Again, please remove references to `unittest`

Reply:
Fixed